public/apps/configuration/constants.tsx

```diff
@@ -156,3 +156,7 @@ export enum ToolTipContent {
   DocumentLevelSecurity = 'Document-level security lets you restrict a role to a subset of documents in an index.',
   FieldLevelSecurity = 'Field-level security lets you control which document fields a user can see.',
 }
+
+// Name constraint: m is lower limit  and n is upper limit
+export const m = 2;
+export const n = 50;
```

Comment:
`m` and `n` is not very meaningful. And for shared constant the name conversion is all upper case. I would suggest use sth. like `MIN_NUMBER_OF_CHARS_IN_RESOURCE_NAME`

Reply:
Used `MIN_NUMBER_OF_CHARS_IN_RESOURCE_NAME` in place of `m` and `MAX_NUMBER_OF_CHARS_IN_RESOURCE_NAME` in place of `n`